app/views/inquiries/index.html.erb

```diff
@@ -19,15 +19,15 @@
             method="get"
             role="search">
         <div class="form-group">
-          <input class="form-control"
+          <input class="form-control search-input"
                  name="q"
                  placeholder="<%= t("global.search") %>"
                  type="text"
                  value="<%= params[:q] %>">
         </div>
-        <button class="btn btn-default" type="submit">
-          <%= t("global.search") %>
-        </button>
+        <button class="btn-icon zbtn-default" type="submit">
```

Comment:
If you're using BEM this should be a modifier class of the btn-icon pattern, so I would do something like `<button class="btn-icon btn-icon--default" type="submit">`.